clippy_lints/src/enum_variants.rs

```diff
@@ -212,6 +212,11 @@ fn check_variant(cx: &LateContext<'_>, threshold: u64, def: &EnumDef<'_>, item_n
     );
 }
 
+#[must_use]
+fn have_no_extra_prefix(prefixes: &Vec<&str>) -> bool {
```

Comment:
Shouldn't that be

```suggestion
fn have_no_extra_prefix(prefixes: &[&str]) -> bool {
```

I'm a bit surprised this passes dogfood.